PhoenixCrypto/PhoenixCrypto/Classes/NativeWeakRandom.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import CryptoKit
+import os.log
+
+fileprivate var log = Logger(
+	subsystem: Bundle.main.bundleIdentifier!,
+	category: "NativeWeakRandom"
+)
+
+@objc
+public class NativeWeakRandom: NSObject {
+
+    @objc
+    private class func toByteArr(i: UInt64) -> [UInt8] {
+        let count = MemoryLayout<UInt64>.size
+        var _i = i
+        let bytePtr = withUnsafePointer(to: &_i) {
+            $0.withMemoryRebound(to: UInt8.self, capacity: count) {
+                UnsafeBufferPointer(start: $0, count: count)
```

Comment:
I'd feel better if the `UnsafeBufferPointer` didn't escape the `withUnsafePointer` lambda:

```swift
return withUnsafePointer(to: &_i) {
  $0.withMemoryRebound(to: UInt8.self, capacity: count) {
    [UInt8](UnsafeBufferPointer(start: $0, count: count))
  }
}
```